src/server/updater/sparkleupdater.mm

```diff
@@ -187,7 +187,14 @@ - (BOOL)supportsGentleScheduledUpdateReminders {
 }
 
 void SparkleUpdater::onUpdateFound() {
-    if (isVersionSkipped(versionInfo().fullVersion())) return;
+    if (isVersionSkipped(versionInfo().fullVersion())) {
+        LOG_INFO(KDC::Log::instance()->getLogger(), "Version " << versionInfo().fullVersion().c_str() << " is skipped.");
+        return;
+    }
+    if ([d->updater sessionInProgress]) {
+        LOG_INFO(KDC::Log::instance()->getLogger(), "A session is already in progress...");
```

Comment:
Maybe a word about the fact the application is already **updating** itself?

Reply:
Done d7a9ea3030ec854e949b671630d16732b74a9ccc
We are waiting for user input or for the end of an ongoing installation

Reply:
Sorry, I was not precise enough: I thought the log message itself could  mention the ongoing update process, to ease off debugging.

Reply:
Done. I have removed the comment, no need to say the same thng twice